GooglePayment/src/main/java/com/braintreepayments/api/models/GooglePaymentRequest.java

```diff
@@ -213,6 +214,17 @@ public GooglePaymentRequest environment(String environment) {
         return this;
     }
 
+    /**
+     * ISO 3166-1 alpha-2 country code where the transaction is processed. This is required for
+     * merchants based in European Economic Area (EEA) countries.
+     * @param countryCode
+     * @return {@link GooglePaymentRequest}
+     */
+    public GooglePaymentRequest setCountryCode(String countryCode) {
```

Comment:
Just confirming - if a merchant were to enter an alpha 3 country code, would Google return an error or would our API return a validation error?

Reply:
This would be a case where the error should be directly from Google, instead of an error from our API.